.pre-commit-config.yaml

```diff
@@ -34,3 +34,4 @@ repos:
         language: system
         pass_filenames: false
         always_run: true
+        args: [ "--cov-fail-under=95" ]
```

Comment:
1. Do we need to specify it here, since it is already in pyproject.toml? 
2. Why is it 95? In the pyproject.toml on main we have 100

Reply:
While developing you may want to have a lower coverage, so you can easily commit intermediate versions.

Reply:
Makes sense :)